src/main/index.ts

```diff
@@ -10,12 +10,15 @@ import * as LensExtensionsCommonApi from "../extensions/common-api";
 import * as LensExtensionsMainApi from "../extensions/main-api";
 import { getDi } from "./getDi";
 import startMainApplicationInjectable from "./start-main-application/start-main-application.injectable";
+import shouldStartHiddenInjectable from "./electron-app/features/should-start-hidden.injectable";
 
 const di = getDi();
 
+const shouldStartHidden = di.inject(shouldStartHiddenInjectable);
+
 const startApplication = di.inject(startMainApplicationInjectable);
 
-void startApplication();
+void startApplication(!shouldStartHidden);
```

Comment:
This seems strange, why is it not just a dependency of `startMainApplication`?

Reply:
It feels more natural to be an argument.